src/components/game.js

```diff
@@ -36,21 +36,152 @@ class Game extends React.Component {
     const board = [];
     for (let i = 0; i < data.site.siteMetadata.size; i += 1) {
       board[i] = new Array(data.site.siteMetadata.size);
+      for (let j = 0; j < data.site.siteMetadata.size; j += 1) {
+        board[i][j] = 0;
+      }
     }
     this.state = {
       board,
     };
+    this.handleKeyDown = this.handleKeyDown.bind(this);
```

Comment:
nie musisz robić bind(this) jeżeli `handleKeyDown` to arrow function (`handleKeyDown = (event) => {`)

Reply:
zrobione